docs/intro/contributing.txt

```diff
@@ -115,6 +115,9 @@ Download the Django source code repository using the following command:
 
     $ git clone git@github.com:YourGitHubName/django.git
 
+You might want to use the ``--depth 1`` argument to ``git clone`` to reduce
+data transfer, as it avoids downloading all of Django's history.
```

Comment:
I'm adding this to ease setup on low bandwidth connections. I think it needs more explanation in a 'note' box.